apps/store/themes/store/js/list_assets.js

```diff
@@ -490,114 +490,116 @@ function resolveDomain() {
+                    var elementRef = document.getElementById(path);
+                    // check wheather its leafnode or not
+                    if ($(elementRef).attr("children") == "true") {
+                        var taxaSub = [];
+                        $.ajax({
+                            url: caramel.context + '/apis/taxonomies?terms=' + path + "/children" + resolveDomain(),
+                            type: 'GET',
+                            async: false,
+                            headers: {
+                                Accept: "application/json"
+                            },
+                            success: function (data) {
+                                var children;
+                                try {
+                                    children = Array.isArray(data[0].children);
+                                } catch (e) {
 
-                            if (children) {
-                                data[0].id = data[0].elementName;
-                                for (var i = 0; i < data[0].children.length; i++) {
-                                    data[0].children[i].id = data[0].elementName + "/" + data[0].children[i].elementName;
                                 }
-                            } else {
-                                for (var i = 0; i < (data.length); i++) {
-                                    data[i].id = $(elementRef).attr("id") + "/" + data[i].elementName;
-                                    if (data[i].text == "") {
-                                        data[i].text = data[i].elementName;
+
+                                if (children) {
+                                    data[0].id = data[0].elementName;
+                                    for (var i = 0; i < data[0].children.length; i++) {
+                                        data[0].children[i].id = data[0].elementName + "/" + data[0].children[i].elementName;
+                                    }
+                                } else {
+                                    for (var i = 0; i < (data.length); i++) {
+                                        data[i].id = $(elementRef).attr("id") + "/" + data[i].elementName;
+                                        if (data[i].text == "") {
+                                            data[i].text = data[i].elementName;
+                                        }
                                     }
                                 }
-                            }
-                            taxaSub = data;
+                                taxaSub = data;
 
-                            //since we have "/" in variable name, jquery selector cant select that element
-                            var currentElement = document.getElementById(path);
-                            createHTMLFromJsonSub(taxaSub, $(currentElement));
+                                //since we have "/" in variable name, jquery selector cant select that element
```

Comment:
This entire code block has been refactored with multiple taxonomy. Its not available anymore.